pages/pricing.tsx

```diff
@@ -29,7 +29,7 @@ const pricingPlan = [
 		pricing: undefined, // will be populated by formula
 		features: ['Access of all features', 'Direct support through Slack', 'Free-of-cost setup assistance'],
 		buttonText: 'Start your 30 day trial',
-		link: '/api/auth/signin?callbackUrl=https%3A%2F%2Fvibinex.com%2Fpricing%2F' // temp. adding login page link, need to replace it with payment link
+		link: '/u'
```

Comment:
The comment will still be valid here right? Why did you remove it?

Reply:
The button text says "Start your 30 day trial". We might not necessarily change the destination of the button click to a payment link. I feel that taking the user to the user profile (or login page and then the user profile page if he/she is not logged in) is actually a good way to handle this button click.